docs/sources/project/test-and-docs.md

```diff
@@ -169,7 +169,7 @@ To run the same test inside your Docker development container, you do this:
 
     root@5f8630b873fe:/go/src/github.com/docker/docker# TESTFLAGS='-run ^TestBuild$' hack/make.sh
 
-## If test under Boot2Docker fail do to space errors
+## If test under Boot2Docker fail due to disk space errors
```

Comment:
Yowza and I spotted another problem test > tests

"If tests under Boot2Docker fail due to disk space errors"

Can you add that to your fix? I'll LGTM it.  Thanks for the catch --- small or large problems we want 'em all!